R/install-git.r

```diff
@@ -19,11 +21,16 @@
 #' install_git("git://github.com/hadley/stringr.git", branch = "stringr-0.2")
 #'}
 install_git <- function(url, subdir = NULL, branch = NULL, args = character(0),
-                        ...) {
+                        force = FALSE, ...) {
   if (!missing(args))
     warning("`args` is deprecated", call. = FALSE)
 
   remotes <- lapply(url, git_remote, subdir = subdir, branch = branch)
+
+  if (!isTRUE(force)) {
```

Comment:
Also this begs the question what the definition of `isFALSE()` should be. `isTRUE()` is defined as `identical(TRUE, x)`, so anything _but_ `TRUE` is `FALSE`. So we do not want `identical(FALSE, x)` but `!identical(TRUE, x)`, so `isFALSE()` is a bit of a misnomer.

It is a small difference so I am happy to define `isFALSE` as `!identical(TRUE, x)`, but it a slightly inaccurate name.